app/views/layouts/application.html.erb

```diff
@@ -3,12 +3,16 @@
   <head>
     <title>On Blast</title>
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
-
   <body>
+<<<<<<< HEAD
```

Comment:
You have some leftover junk from your merge conflicts, please remove the git syntax